android/src/fr/squirrel/pdfview/PdfViewer.java

```diff
@@ -38,6 +38,7 @@ public class PdfViewer extends TiUIView implements OnErrorListener, LinkHandler
 
     PDFView pdfView;
     int space = 20;
+    int pageNumber = 0;
```

Comment:
In case this is unclear: Page numbers are indices, so index 0 = page 1

Reply:
added a small suggestion for the Readme. After that I can merge and release it. Working fine